src/puya/ir/context.py

```diff
@@ -16,7 +16,7 @@ class IRBuildContext(CompileContext):
     module_awsts: Mapping[str, awst_nodes.Module]
     subroutines: dict[awst_nodes.Function, Subroutine]
     embedded_funcs: Sequence[awst_nodes.Function] = attrs.field()
-    contract: awst_nodes.ContractFragment | None = attrs.field(default=None)
+    contract: awst_nodes.ContractFragment | None = None
```

Comment:
Curious, is there a reason for these changes other than stylistic preference?

Reply:
yeah just harder to read and looks like something more complicated is going on, when it's just a default